objects/status/src/object_status.c

```diff
@@ -7,4 +7,17 @@
 
 /// \file objects/status/src/object_status.c
 
-#include "object_status.h"
\ No newline at end of file
+#include "object_status.h"
+
+status_t status_init(void)
+{
+    status_t new_status;
+
+    new_status = OFFLINE;
+    return new_status;
+}
+
+void status_destroy(status_t status_to_destroy)
+{
+    (void) status_to_destroy;
+}
```

Comment:
useless to destroy status

Reply:
Need the function destroy for modularity and generality